src/nodes/display/ViewportSharedTextureNode.js

```diff
@@ -6,6 +6,13 @@ import { FramebufferTexture } from '../../textures/FramebufferTexture.js';
 
 let _sharedFramebuffer = null;
 
+/**
+ * `ViewportTextureNode` creates an internal texture for each texture instance. This module
+ * shares a texture across all instances of `ViewportSharedTextureNode`. It should
```

Comment:
@sunag One question about the texture sharing in `ViewportSharedTextureNode`:

Right now, the module only shares the instance of `FramebufferTexture` across all nodes. However, for each node, `updateBefore()` is called and thus `copyFramebufferToTexture()` is executed which overwrites the data in the shared texture instance. Is that correct?

One could expect the copy only happens once per frame no matter how many `ViewportSharedTextureNode` objects are used.  

The current behavior seems intended because `ViewportSharedTextureNode` returns `this` in its overwritten `updateReference()` method. Without this bit (meaning by using the default implementation from `TextureNode`), `NodeFrame` should execute `updateBefore()` once per frame for _all_ `ViewportSharedTextureNode` instances.